src/prefect/agent/ecs/agent.py

```diff
@@ -508,16 +497,22 @@ def get_run_task_kwargs(
             container = {"name": "flow"}
             container_overrides.append(container)
 
+        # Set flow run command
+        container["command"] = ["/bin/sh", "-c", get_flow_run_command(flow_run)]
```

Comment:
Should this be 
```suggestion
        # Set flow run command
        container.setdefault("command", ["/bin/sh", "-c", get_flow_run_command(flow_run)])
```

Reply:
We don't let users in other execution environments override this, I don't think we shouldn't here. If a user wants to extend what happens at startup time they can set an `entrypoint` (either as part of the image, or via the task definition).

Reply:
Perhaps with 

```python
if get_flow_run_command(flow_run) not in container["command"]:
    warnings.warn("...")
```

Reply:
Alrighty. Perhaps a warning or exception if it's already set then?

Reply:
Sure. Can add tomorrow.